ThirdPartyAdapters/vungle/settings.gradle

```diff
@@ -1 +1,13 @@
+rootProject.name = 'vungle'
```

Comment:
We might have needed this for our Test app project, do we know if this could cause issues to anyone picking our vungle module and this settings file as-is , will it interfere with their root project
[Open question that am looking for suggestions]:
On same lines can we avoid to not have this change as well as one related to sdkSrcDependency  being send upstream , while continue to keep it in our develop brach in forked repo, is there a way we can we mark certain files to be ignored in upstream PRs

Reply:
Done. Removed this line.